policies/apis/gith-branch-protection.yml

```
@@ -0,0 +1,47 @@
+attest:
+  cocosign:
+    policies:
+    - name: github-branch-protection
+      enable: true
+      modules:
+      - name: github-branch-protection
+        type: verify-artifact
+        enable: true
+        input:
+          signed: false
+          rego:
+            args:
+              api_token: '{{ .Args.Token }}'
+              owner: '{{ .Args.Owner }}'
+              repo: '{{ .Args.Repo }}'
+              branch: '{{ .Args.Branch }}'
+              branch_protection_rules:
+                required_pull_request_reviews.dismiss_stale_reviews: true
+                required_pull_request_reviews.require_code_owner_reviews: false
+                required_pull_request_reviews.require_last_push_approval: false
+                required_pull_request_reviews.required_approving_review_count: 1
+                required_pull_request_reviews.bypass_pull_request_allowances.users: []
+                required_pull_request_reviews.bypass_pull_request_allowances.teams: []
+                required_pull_request_reviews.bypass_pull_request_allowances.apps: []
+                required_pull_request_reviews.dismissal_restrictions.users: []
+                required_pull_request_reviews.dismissal_restrictions.teams: []
+                required_pull_request_reviews.dismissal_restrictions.apps: []
```

@dn-scribe will it work?

```suggestion
              branch_protection_rules:
                required_pull_request_reviews:
                  dismiss_stale_reviews: true
                  equire_code_owner_reviews: false
                  require_last_push_approval: false
                  required_approving_review_count: 1
                  bypass_pull_request_allowances.users: []
                  bypass_pull_request_allowances.teams: []
                  bypass_pull_request_allowances.apps: []
                  dismissal_restrictions.users: []
                  dismissal_restrictions.teams: []
                  dismissal_restrictions.apps: []
```